gyp/pylib/gyp/msvs_emulation.py

```diff
@@ -67,7 +74,8 @@ def EncodeRspFileList(args):
         program = call + " " + os.path.normpath(program)
     else:
         program = os.path.normpath(args[0])
-    return program + " " + " ".join(QuoteForRspFile(arg) for arg in args[1:])
+    return (program + " " +
```

Comment:
Looks different lint rules result in this. Should we skip gyp lib in this repo or unify the rule ?

Reply:
Put the + at the beginning of the next line.

Reply:
@cclauss 

Should we add this rule in https://github.com/nodejs/gyp-next ?

Also should we disable flake8 test for this repo since gyp sources is an upstream project ?